src/main/webapp/WEB-INF/tags/instructor/results/filterPanel.tag

```diff
@@ -61,6 +61,11 @@
                           </label>
                       </div>
                   </div>
+                  <div data-toggle="tooltip" title="Indicate missing responses">
+                      <div class="checkbox padding-top-0 min-height-0">
+                              <input type="checkbox" onchange="this.form.submit()" id="indicate-missing-responses-checkbox" value="true" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_INDICATE_MISSING_RESPONSES%>"<c:if test="${filterPanel.missingResponsesShown}"> checked</c:if>> Indicate Missing Responses
```

Comment:
The indentation step should be 4 spaces.
Other than that, the line is too long. See if you can break it at different attributes. You can look at `feedbackSessionActions.tag` to see how it's done.